app/helpers/banner_helper.rb

```diff
@@ -3,6 +3,10 @@
 require "yaml"
 
 module BannerHelper
+  def orcid_available?
+    HealthMonitor.check[:results].find { |service| service[:name] == "OrcidApiStatus" }[:status] == "OK"
```

Comment:
Is this value cached? 

It might worth to confirm otherwise we'll be making an API call to ORCID every time this method is called (which seems to be 4 times (one in application.html.erb, one in _user.html.erb, and one or two in user/show.html.erb)

Reply:
We were calling this twice on the user page and once on the homepage.  Memoizing the result of the helper method helped address this, thanks @hectorcorrea for pairing.